core/kazoo_documents/src/kzd_freeswitch.erl

```diff
@@ -52,11 +52,14 @@
 
 -include("kz_documents.hrl").
 
+-type data() :: kz_proplist().
```

Comment:
We're in kazoo_documents & all other modules define a `doc/0` type. How about we continue doing this?